### PR TITLE
Dev TLS Cert and port mapping changes.

### PR DESCRIPTION
**Pull request by DreamWver**
Created new images for proxy and nicsweb with embedded tls certs.

### CODE REVIEW

**Review by comeara — CHANGES_REQUESTED** [review]
It looks like Apache is still trying to redirect to port 3443. I think the port change requires a corresponding change to Apache.

**Comment by DreamWver** [comment]
Good Catch. I'll make the change to the nics.conf.sample over on nics-www.

File: docker-compose.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ services:
     image: 618209257270.dkr.ecr.us-west-2.amazonaws.com/nics/proxy:latest
     ports:
       - "3000:80"
```
Comment (comeara) [inline docker-compose.yml]: I suggest you also move http to 80 rather than 3000 if you're going to use 443 for https.

**Comment by comeara** [comment]
I retested this today and it is still broken due to the fact that the proxy:dev-configured and web:latest images have not been updated.

**Comment by DreamWver** [comment]
I updated the proxy:dev-configured and web:latest images. And changed the port mapping on the proxy as you requested. Also made one more port mapping change that was plaguing us due to a service conflict.

**Review by sneha-nagole — APPROVED** [review]

**Review by comeara — APPROVED** [review]

**Pull request merged**